CHANGELOG_PENDING.md

```diff
@@ -13,4 +13,9 @@
 
 - Fix #478: New API Pulumi.withOptions adds support for stack transformations
 
+- [sdk] PulumiTest and PulumiTestInternal replaces DeploymentTests, 
```

Comment:
not sure if we should consider this breaking change or not, but the public API for tests has changed significantly

Reply:
Yes. I'm on the fence here, I guess it is not breaking since we did not sufficiently advertise how to use the old API. So perhaps we just announce this as the new API release.

Suggest something like this.

```
- [sdk] #553: Added `com.pulumi.test` package with new idomatic Java API for [unit testing Pulumi programs]
(https://www.pulumi.com/docs/guides/testing/#unit-testing) against mocks. See ([example](<url-to-example here>)) of  using the new API.

```